tls/extensions/s2n_client_alpn.h

```diff
@@ -20,6 +20,8 @@
 #include "stuffer/s2n_stuffer.h"
 
 extern const s2n_extension_type s2n_client_alpn_extension;
+bool s2n_client_alpn_should_send(struct s2n_connection *conn);
+int s2n_select_supported_protocol(struct s2n_connection *conn, struct s2n_blob *protocol_preferences, struct s2n_stuffer *extension);
 
```

Comment:
The fact that this method doesn't match the module name makes me wonder if there's a better place to put it.